### ISSUE TITLE
Filtering based on content

### ISSUE THREAD

**Issue by quotengrote** [tborychowski/miniflux-filter#3]
Hi,

is it possible to filter a message based on keywords in the Content?

Wishes
mg

**Comment by tborychowski** [comment]
Great idea. I will add that!

**Comment by tborychowski** [comment]
Just released v1.3.0 with this feature.
Enjoy! :-)

#### example usage:
```yml
filters:
  - url: 'my-rss-url.com'   # matches url of the article, rss feed, or the site
    content: 'keyword'       # a keyword of a phrase to look for in articles' body/content section
```

**Comment by quotengrote** [comment]
Thank you very much! 
Just testing it, working great.
